test/functional/sqlsrv/sqlsrv_ae_insert_money.phpt

```diff
@@ -0,0 +1,77 @@
+--TEST--
+Test for inserting and retrieving encrypted money types data
+Bind params using sqlsrv_prepare without any sql_type specified
+--SKIPIF--
+<?php require('skipif_versions_old.inc'); ?>
+--FILE--
+<?php
+include 'MsCommon.inc';
+include 'AEData.inc';
+include 'MsSetup.inc';
+
+$dataTypes = array( "smallmoney", "money" );
+$conn = ae_connect();
+
+foreach ( $dataTypes as $dataType ) {
+    echo "\nTesting $dataType: \n";
+    $success = true;
+    
+    // create table
+    $tbname = GetTempTableName( "", false );
+    $colMetaArr = array( new columnMeta( $dataType, "c_det" ), new columnMeta( $dataType, "c_rand" ));
+    create_table( $conn, $tbname, $colMetaArr );
+    
+    // insert a row
+    $inputValues = array_slice( ${explode( "(", $dataType )[0] . "_params"}, 1, 2 );
+    $r;
+    $stmt = insert_row( $conn, $tbname, array( $colMetaArr[0]->colName => $inputValues[0], $colMetaArr[1]->colName => $inputValues[1] ), $r );
+    
+    if ( $keystore == "none" )
+    {
```

Comment:
Perhaps add a function to MsCommon.inc to return a boolean that indicates if AE is on (i.e. if $keystore is "none") then in this case you don't really need to include 'MsSetup.inc' everywhere